example/linux/systemd/Talawa-api.sh

```
@@ -0,0 +1,160 @@
+  echo "Error: Unable to read 'engines.node' from package.json. Exiting." | tee -a "$LOG_FILE"
+  exit 1
+fi
+
+# Remove 'v' prefix if present, e.g. "v20.18.0" -> "20.18.0"
+INSTALLED_NODE_VERSION=$(node -v 2>/dev/null | sed 's/^v//')
+
+echo "Installed Node.js version: $INSTALLED_NODE_VERSION" | tee -a "$LOG_FILE"
+echo "Target Node.js version: $TARGET_NODE_VERSION" | tee -a "$LOG_FILE"
+
+if [ "$INSTALLED_NODE_VERSION" != "$TARGET_NODE_VERSION" ]; then
+  echo "Error: Node.js version mismatch. Found $INSTALLED_NODE_VERSION, need $TARGET_NODE_VERSION". Exiting." | tee -a "$LOG_FILE"
```

Shellcheck (0.10.0)

[warning] The surrounding quotes actually unquote this. Remove or escape them.

(SC2027)